packages/react-composites/src/composites/CallComposite/hooks/useAdaptedSelector.ts

```diff
@@ -98,7 +98,8 @@ const memoizeCalls = memoizeOne((call?: CallState): { [key: string]: CallState }
 
 const adaptCompositeState = (compositeState: CallAdapterState): CallClientState => {
   return memoizeState(
-    compositeState.userId,
+    // xkcd: FIXME
```

Comment:
dckx //

Reply:
Heh, I made this change halfway through the refactor, and knew it was easy to miss ('cause the types are OK, just unnecessary casting).

And did I forget... :)